src/lib/registration/ui/not_installed_products_dialog.rb

```diff
@@ -126,10 +126,7 @@ def install_products
 
             result = Yast::Pkg.PkgCommit(0)
             # success?
-            if result && result[1].empty?
-              Yast::PackagesUI.show_update_messages(result)
```

Comment:
Please add it as commented out with a reason, it will help when doing other backports later.